pubspec.yaml

```diff
@@ -43,8 +43,8 @@ dependencies:
   device_info_plus: ^11.2.0
   drift: ^2.23.0
   file_picker: ^10.1.2
-  firebase_core: ^3.3.0
-  firebase_messaging: ^15.0.1
+  firebase_core: ^4.0.0
+  firebase_messaging: ^16.0.0
```

Comment:
nit:

    deps: Upgrade firebase_core, firebase_messaging to latest
    
    This commit is result of following commands:

That latter line is a lot like how we write summary lines for commits. But here in the commit-message body, it should instead be a normal English sentence:

> This commit is the result of the following commands:

More generally: those summary lines are like the headlines of news articles, and the unusual, compressed syntax we use there is basically a form of "headlinese":
https://en.wikipedia.org/wiki/Headline#Headlinese

But then the body of a commit message should be written in prose, as normal English sentences (much like how the body of a news article is written in prose, and not in headlinese).